.travis.yml

```diff
@@ -4,7 +4,10 @@ go:
   - 1.7.x
 sudo: false
 install:
-  - go get -v github.com/Masterminds/glide
+  - GLIDE_TAG=v0.12.3
+  - GLIDE_PKG=github.com/Masterminds/glide
+  - go get -v $GLIDE_PKG
```

Comment:
Good call.  I forget to put `-d` on there so it's download only.  I'll do that now.